webapp/src/pages/404.tsx

```diff
@@ -76,7 +73,7 @@ export const NotFound = ({
       ) : (
         <></>
       )}
-      <h1 style={{ color: 'white' }}></h1>
+      <h1></h1>
```

Comment:
Does this get injected with something, or is it stylized to just have a bar or something? Confused why it's in here

Reply:
removed!